docs/release_notes/plugin_api/0.1.x.rst

```diff
@@ -0,0 +1,8 @@
+Plugin API 0.1.alpha Release Notes
```

Comment:
I think `0.0.z.rst` is right.  I'm generally +0 on using `x.y.z` versus `x.y` only. bugfixes may be infrequent unlike platform. Maybe having the z bit is also good. I'm not sure. Let's probably leave it at x.y.z for now.